modules/claims_api/lib/bd/bd.rb

```diff
@@ -80,20 +81,21 @@ def generate_upload_body(claim:, pdf_path:, file_number: nil)
     #
     # @return Faraday client
     def client
-      base_name = if !Settings.claims_api.evss_container&.auth_base_name.nil?
-                    "#{Settings.claims_api.evss_container.auth_base_name}/services"
+      base_name = if !Settings.claims_api.benefits_documents&.host.nil?
+                    "#{Settings.claims_api.benefits_documents.host}/services"
```

Comment:
this value only exists in our production environment specific settings file. we'll default the rest to the settings.yml for all environments. so with this change, we'll be calling sandbox from dev-api.va.gov and staging-api.va.gov, which defeats the purpose of mocking sandbox.

I happened to catch this last week because I was curious why I used evss_container value, and it's because they don't exist. This is set to EVSS because those aligned with what we needed to get the search working in production (this was essentially like a "patch" when we turning it on in production). 

I agree that they should be more in line with the changes you have above - this will require PRs to each environment (minus production values.yml).